azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -44,29 +44,27 @@
     public static final String FUNCTION_APP_UPDATE = "Updating Azure Function App...";
     public static final String FUNCTION_APP_UPDATE_DONE = "Successfully updated Azure Function App ";
 
-    public static final String MS_DEPLOY = "msdeploy";
-    public static final String FTP = "ftp";
-
     //region Properties
 
     /**
      * Deployment type to deploy Web App. Supported values:
      * <ul>
      * <li>msdeploy</li>
      * <li>ftp</li>
+     * <li>zip</li>
      * </ul>
      *
      * @since 0.1.0
      */
-    @Parameter(property = "functions.deploymentType", defaultValue = MS_DEPLOY)
```

Comment:
So we don't need a default value?

Reply:
If the annotation can assign a default value, then using it is preferred than write extra code. An example, when you define a variable, you want to initialize it immediately like this:
int i = 100;
But not:
int i;
...
i = 100;

Reply:
updated